src/Microsoft.DotNet.ProjectModel/Project.cs

```diff
@@ -84,6 +84,8 @@ public string ProjectDirectory
 
         public bool IsTestProject => !string.IsNullOrEmpty(TestRunner);
         
+        public bool OverrideIsRunnable { get; set; }
```

Comment:
Ah, I see what you mean now.  I thought you meant that the template (and users) need to set it in the project.json file.

fixed.

---
In reply to: [71798696](https://github.com/dotnet/cli/pull/3911#discussion_r71798696) [](ancestors = 71798696,71797149,71797075)